packages/api-demo/package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "scripts": {
     "serve:no-browser": "npm run serve -- --no-browser",
-    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/containerjs-bundle.js --mount=/resources/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
+    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api:node_modules/containerjs-api-browser/build/dist/containerjs-api.js --mount=/resources/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
```

Comment:
I wondered about that, but then I noticed you'd added an option to ssf-openfin to generate a notification template, so I thought you'd decided to do it that way instead. But yes, I guess we will still need to map a path to wherever that file gets generated...